receipt is missing on error event (#3129)
ethereum/web3.js: docs/web3-eth.rst

- ``"transactionHash"`` returns ``String``: Is fired right after the transaction is sent and a transaction hash is available.
- ``"receipt"`` returns ``Object``: Is fired when the transaction receipt is available.
- ``"confirmation"`` returns ``Number``, ``Object``: Is fired for every confirmation up to the 12th confirmation. Receives the confirmation number as the first and the :ref:`receipt <eth-gettransactionreceipt-return>` as the second argument. Fired from confirmation 0 on, which is the block where its minded.

> - ``"error"`` returns ``Error`` and ``Object|undefined``: Is fired if an error occurs during sending. If a out of gas error, the second parameter is the receipt.

Reviewer: ```suggestion
``"error"`` returns ``Error`` and ``Object|undefined``: Is fired if an error occurs during sending. If the transaction was rejected by the network with a receipt, the second parameter will be the receipt.
```
